pkg/app/master/commands/debug/cli.go

```diff
@@ -16,11 +16,9 @@ const (
 	Usage = "Debug the target container image from a debug container"
 	Alias = "dbg"
 
-	FlagDebugImage    = "debug-image"
-	FlagDebugImageCmd = "debug-image-cmd"
+	FlagDebugImage = "debug-image"
 
-	DefaultDebugImage    = "nicolaka/netshoot"
-	DefaultDebugImageCmd = "/bin/bash"
```

Comment:
what about the default command now?

Reply:
The one from the image is used, I originally used bash if nothing was provided, but if you leave this field blank, then you just get the image's default which is even better in most cases.